build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml

```diff
@@ -121,104 +125,132 @@ objects:
             successThreshold: 1
             timeoutSeconds: 2
           resources:
-            requests:
-              cpu: 100m
-              memory: 40Mi
             limits:
               cpu: 100m
               memory: 60Mi
-        - name: etcd
-          image: quay.io/coreos/etcd:latest
-          imagePullPolicy: Always
-          resources:
             requests:
               cpu: 100m
-              memory: 20Mi
-            limits:
-              cpu: 100m
-              memory: 30Mi
-          env:
-          - name: ETCD_DATA_DIR
-            value: /etcd-data-dir
-          command:
+              memory: 40Mi
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+          - mountPath: /var/run/cluster-api-apiserver
+            name: cluster-apiserver-certs
+            readOnly: true
+        - command:
           - /usr/local/bin/etcd
           - --listen-client-urls
           - http://0.0.0.0:2379
           - --advertise-client-urls
           - http://localhost:2379
-          ports:
-          - containerPort: 2379
-          volumeMounts:
-          - name: etcd-data-dir
-            mountPath: /etcd-data-dir
-          readinessProbe:
+          env:
+          - name: ETCD_DATA_DIR
+            value: /etcd-data-dir
+          image: quay.io/coreos/etcd:latest
+          imagePullPolicy: Always
+          livenessProbe:
+            failureThreshold: 3
             httpGet:
-              port: 2379
               path: /health
-            failureThreshold: 1
+              port: 2379
+              scheme: HTTP
             initialDelaySeconds: 10
             periodSeconds: 10
             successThreshold: 1
             timeoutSeconds: 2
-          livenessProbe:
+          name: etcd
+          ports:
+          - containerPort: 2379
+            protocol: TCP
+          readinessProbe:
+            failureThreshold: 1
             httpGet:
-              port: 2379
               path: /health
-            failureThreshold: 3
+              port: 2379
+              scheme: HTTP
             initialDelaySeconds: 10
             periodSeconds: 10
             successThreshold: 1
             timeoutSeconds: 2
+          resources:
+            limits:
+              cpu: 100m
+              memory: 30Mi
+            requests:
+              cpu: 100m
+              memory: 20Mi
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+          - mountPath: /etcd-data-dir
+            name: etcd-data-dir
```

Comment:
Was this file updated with the results of an actual object get? Looks like a lot of defaults might have been added in. In this case I'd also be inclined to stick to just defining what we care about and letting the server apply it's defaults.